src/CMakeLists.txt

```diff
@@ -143,10 +143,19 @@ else()
     set(ZTESTSOURCES ${ZTESTSOURCES} recv-pcap.c)
 endif()
 
-configure_file(topt.ggo.in ${CMAKE_BINARY_DIR}/src/topt.ggo @ONLY)
-configure_file(zbopt.ggo.in ${CMAKE_BINARY_DIR}/src/zbopt.ggo @ONLY)
-configure_file(zitopt.ggo.in ${CMAKE_BINARY_DIR}/src/zitopt.ggo @ONLY)
-configure_file(zopt.ggo.in ${CMAKE_BINARY_DIR}/src/zopt.ggo @ONLY)
+# Set configure time zmap version
+configure_file(topt.ggo.in topt.ggo @ONLY)
```

Comment:
Why did you remove the `${CMAKE_BINARY_DIR}`? You probably broke out of source builds, which will make package maintainers upset.